protos/local/core_interface.proto

```diff
@@ -16,12 +16,28 @@ import "google/protobuf/timestamp.proto";
 import "google/protobuf/duration.proto";
 import "google/protobuf/empty.proto";
 
-// A request as given to [crate::Core::send_activity_heartbeat]
+// A request as given to [crate::Core::record_activity_heartbeat]
 message ActivityHeartbeat {
-    string activity_id = 1;
+    ActivityId activity_id = 1;
     repeated common.Payload details = 2;
 }
 
+message ActivityId {
+    oneof id {
+        bytes task_token = 1;
+        ManualCompletionActivityTaskToken manual_completion_task_token = 2;
+    }
+}
+
+message ManualCompletionActivityTaskToken {
```

Comment:
I thought we could just use a service client to manually complete the activity since we might want to complete manually from a process that's not running a worker.

Reply:
That's a fair point, let's simplify it for now and see if we need manual completion in the core later.